salt/metalk8s/orchestrate/deploy_etcd_on_new_node.sls

```diff
@@ -0,0 +1,81 @@
+{%- set control_plane_ips = salt.saltutil.runner('mine.get', tgt='*', fun='control_plane_ips') %}
+
+{%- if pillar['bootstrap_id'] in control_plane_ips.keys() and control_plane_ips[pillar['bootstrap_id']] %}
+{%- set bootstrap_ip = control_plane_ips[pillar['bootstrap_id']][0] %}
+{%- else %}
+{%- set bootstrap_ip = 'localhost' %}
+{%- endif %}
+
+{%- if pillar['node_name'] in control_plane_ips.keys() and control_plane_ips[pillar['node_name']] %}
+{%- set node_ip = control_plane_ips[pillar['node_name']][0] %}
+{%- else %}
+{%- set node_ip = 'localhost' %}
+{%- endif %}
+
+{#- etcd endpoint of the new node. #}
+{%- set endpoint  = 'https://' ~ node_ip ~ ':2380' %}
+
+Generate etcd certificates on a new node:
+  salt.state:
+    - tgt: {{ pillar['node_name'] }}
+    - saltenv: {{ saltenv }}
+    - sls:
+      - metalk8s.kubeadm.init.certs.etcd-server
+      - metalk8s.kubeadm.init.certs.etcd-peer
+      - metalk8s.kubeadm.init.certs.etcd-healthcheck-client
+    - pillar:
+        repo:
+          online_mode: false
+          local_mode: false
+          host: {{ bootstrap_ip }}
+        registry_ip: {{ bootstrap_ip }}
+
+Refresh the list of etcd nodes:
+  module.run:
+    - mine.update:
+      - mine_functions: metalk8s.get_etcd_endpoint
+
+Deploy the new etcd node:
+  salt.state:
+    - tgt: {{ pillar['node_name'] }}
+    - saltenv: {{ saltenv }}
+    - sls:
+      - metalk8s.bootstrap.etcd
+    - pillar:
+        repo:
+          online_mode: false
+          local_mode: false
+          host: {{ bootstrap_ip }}
+        registry_ip: {{ bootstrap_ip }}
+    - require:
+      - salt: Generate etcd certificates on a new node
+      - module: Refresh the list of etcd nodes
+
+Deploy updated manifests for existing etcd nodes:
```

Comment:
Hum, really not sure about this one.

The more i think about it, the more it seems wrong.
I'll redo tests without it, looks like a wild guess made during our troubleshooting session.

Reply:
I don't think there's a need for this indeed?

Reply:
agree there is no need to touch existing members

Reply:
Indeed, I confirmed that it was useless.